ge/org.osate.ge/src/org/osate/ge/internal/ui/handlers/ShowContentsUtil.java

```diff
@@ -40,16 +41,20 @@
 import org.osate.ge.internal.services.ExtensionRegistryService;
 import org.osate.ge.internal.ui.editor.InternalDiagramEditor;
 import org.osate.ge.internal.util.BusinessObjectProviderHelper;
+import org.osate.ge.internal.util.ContentFilterUtil;
 import org.osate.ge.services.ReferenceBuilderService;
 
+import com.google.common.collect.ImmutableSet;
+
 class ShowContentsUtil {
 	/**
 	 * Adds contents to the selected diagram elements. Adds all children which pass the specified filter.
 	 * @param event is the ExecutionEvent of the handler which provides the active editor.
-	 * @param filter is passed the parent diagram element and the business object of the potential child. If the filter returns true, then an element will be added for the business object.
+	 * @param filters is passed the parent diagram element.  It returns the applicable content filters for the diagram element to
```

Comment:
Should be "its". Elaborate on what the purpose of the argument is. It determines the filters to apply to an element, correct?